package.json

```diff
@@ -30,8 +25,8 @@
     "superagent": "^5.3.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
+    "@babel/cli": "^7.16.0",
+    "@babel/core": "^7.16.0",
```

Comment:
Note: those were first updated in the `client-code-generation` folder.

Why: staying current